src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -4073,7 +4073,7 @@ void M()
     {
         int a = 1;
         int {|Rename:y1|} = C.y;
-        var t = (a, x: y1);
+        var t = (a: a, x: y1);
```

Comment:
>  but if it's something we make diagnostics for, I'm more leery, as people might be upset having a bunch of these appear in their error list when they upgrade VS. 

I think that's what we want.  Users who like to clean up their code will want messages letting them know they can simplify this code once they upgrade past the appropriate version of C#.